src/encoded/upgrade/user.py

```diff
@@ -11,10 +11,11 @@ def user_0_3(value, system):
 @upgrade_step('user', '3', '4')
 def user_3_4(value, system):
     # http://redmine.encodedcc.org/issues/4743
-    value['viewing_groups'] = [
-        'ENCODE3' if v == 'ENCODE' else v
-        for v in value['viewing_groups']
-    ]
+    if 'viewing_groups' in value:
```

Comment:
You shouldn't change anything in this file. The upgrades have already been applied to specific schema versions (current user version is 8) and stored in the database. Changing things here is tampering with the historical record of migrations and won't have any effect on current schema version.

Reply:
Still not sure why there are changes here? You shouldn't touch these.

Reply:
I missed that. My revert went wrong and I misread the diff changes prior to completion. It is fixed